docs/transport/transport.md

```diff
@@ -0,0 +1,61 @@
+# gittuf Remote Transfer Protocol
+
+Last Modified: July 2, 2024
+
+## Introduction
+
+Alongside the `gittuf` binary, gittuf ships with a custom remote transfer
+protocol binary, implementing Git's [remote-helper
+interface](https://git-scm.com/docs/gitremote-helpers). We call this the
+**transport** binary, named `git-remote-gittuf`.
+
+It's an easy way to get started with using gittuf on your repository, as it
+takes care of the following common operations for you:
+
+- Creating RSL entries upon pushing your changes
+- Verifying changes you pull down
```

Comment:
Fetching gittuf metadata when pulling changes

we haven't enabled verification yet fyi